### DISCUSSION TITLE
Allow shutting off screen

### DISCUSSION THREAD

**Discussion by ss8688** [ExpressLRS/ExpressLRS#1573] [General]
The screen is useless to edgetx users，except consuming electricity. So  It is good to add a shutting off screen option.

**Comment by pkendall64** [comment]
I'm assuming you mean on the OLED modules?

**Reply by ss8688** [reply]
Yes

**Comment by xou130** [comment]
Maybe add this for save the oled screen life span. Monochrome OLEDs will burn in like colored ones. [https://hackaday.com/2019/04/23/a-year-long-experiment-in-oled-burn-in/](url)
